docs/source/examples.rst

```diff
@@ -0,0 +1,26 @@
+Examples
+========
+
+
+Early detecting invalid password reset tokens
+---------------------------------------------
+
+When there is need to check if password reset token is still valid without actually resetting the password it is possible
+to approach the problem like so:
+
+.. code-block:: python
+
+    from djoser import serializers
```

Comment:
Since you include imports in your snippet I'd also add other required imports:

``` python
from django.contrib.auth.tokens import default_token_generator
from rest_framework import generics, permissions, status
from rest_framework.response import Response
```